recipes/scheil/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "scheil" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "de43e05fb9fdfb084fa3c7709000cc81c88292eaee24a0ac020c52c032c0c809" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Please add `pip` in the `host` requirements. Also, I don't think `setuptools` is needed here.